project/Build.scala

```diff
@@ -1389,27 +1389,27 @@ object Build {
 
     // FIXME: we do not aggregate `bin` because its tests delete jars, thus breaking other tests
     def asDottyRoot(implicit mode: Mode): Project = project.withCommonSettings.
```

Comment:
We should add `publishArtifact := false` to the dotty root settings so we don't publish an artefact called `dotty` or `scala3`.